apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.html.ts

```diff
@@ -26,99 +26,85 @@ export function Lti13CourseNavigationInstructor({
       </head>
       <body>
         ${Navbar({ resLocals, navPage: 'lti13_course_navigation' })} ${TerminologyModal()}
-        <script>
-          $(() => {
-            $('#connect_course_instance').one('change', () => {
-              $('#saveButton').prop('disabled', false);
-            });
-          });
-        </script>
-
         <main id="content" class="container mb-4">
           <h1>Welcome to PrairieLearn</h1>
           <p>
             To finish the integration for your course, we need to connect
-            <code>${courseName}</code> with a PrairieLearn course instance.
+            <strong>${courseName}</strong> with a PrairieLearn course instance.
           </p>
           <p>
             <button
               type="button"
               class="btn btn-sm btn-info"
-              data-toggle="modal"
-              data-target="#terminology-modal"
+              data-bs-toggle="modal"
+              data-bs-target="#terminology-modal"
             >
               New here? Learn about our terminology
```

Comment:
This button is so bright it feels like the most important thing on the page, whereas I think it's actually very unimportant. Can we just delete it? Or if you really want to keep it, shift it to the very bottom of the page as a link (not a button) like:
```
<p>
  Is something unclear above? <a href="#" data-bs-toggle="modal" data-bs-target="#terminology-modal">Learn about our terminology.</a>
</p>
```

Reply:
I kept the original wording but moved the button to the bottom and styled it as a link in 633032758. New output:

![image](https://github.com/user-attachments/assets/0775e6d4-c5d8-4b16-ba7b-bae898bab7f2)

Reply:
Oops, just saw the "our" note. Updated to say `Learn about PrairieLearn terminology` in 944b77425